test/app/script-tag-integration.html

```diff
@@ -0,0 +1,49 @@
+<!doctype html>
```

Comment:
If we don't change anything, it will be automatically added when we deploy it next.